modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheMapEntry.java

```diff
@@ -1779,11 +1779,8 @@ else if (ttl != CU.TTL_ZERO)
             if (metrics &&
                 updateRes.outcome().updateReadMetrics() &&
                 cctx.statisticsEnabled() &&
-                needVal) {
-                // PutIfAbsent methods must not update hit/miss statistics.
```

Comment:
Pay attention to method GridCacheMapEntry.innerUpdateLocal() too, please.

Reply:
@NSAmelchev Done